tests/hash.test.ts

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+
+    Test that create hash.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as assert from 'assert';
+import * as hm from '../src/modules/common/Hash'
+const hms = hm.default;
+
+describe('Hash', () => {
+    // Buffer has the same content. However, when printed with hex strings,
+    // the order of output is different.
+    // This was treated to be the same as D language.
+    it('Test of reading and writing hex string', () => {
+        // Read from hex string
+        let h =
+            new hms.Hash('0x5d7f6a7a30f7ff591c8649f61eb8a35d034824ed5cd252c2c6f10cdbd223671' +
+                    '3dc369ef2a44b62ba113814a9d819a276ff61582874c9aee9c98efa2aa1f10d73');
+
+        // Check
+        assert.equal(h.toHexString(),
+            '0x5d7f6a7a30f7ff591c8649f61eb8a35d034824ed5cd252c2c6f10cdbd2236713d' +
+            'c369ef2a44b62ba113814a9d819a276ff61582874c9aee9c98efa2aa1f10d73');
+    });
+
+    it('Test of hash', () => {
+        // Create Hash class
+        let h = new hms.Hash();
+
+        // Hash
+        h.hash(Buffer.from("foo"));
+
+        // Check
+        assert.equal(h.toHexString(),
+            '0x8d8fd1dd408231aba0d7bbc59d2d98bb14c724843720e13707aeb43dfb23e7c16' +
+            'd5d1854b343ffc2d6cab73b7ad9519dd73f53a6566aa4846b3e9de6302300ca');
```

Comment:
```suggestion
        // Test with https://tools.ietf.org/html/rfc7693#appendix-A
        h.hash(Buffer.from("abc"));
        assert.equal(h.toHexString(),
            '0x239900d4ed8623b95a92f1dba88ad31895cc3345ded552c22d79ab2a39c5877'
            'dd1a2ffdb6fbb124bb7c45a68142f214ce9f6129fb697276a0d4d1c983fa580ba');
```

It's better to use the RFC for this.

Reply:
I changed